mfa_duo.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/duosecurity/duo_api_golang"
+	"github.com/duosecurity/duo_api_golang/authapi"
+	"github.com/pkg/errors"
+	yaml "gopkg.in/yaml.v2"
+)
+
+const allow = "allow"
```

Comment:
These constants are global, name them specific to the Duo MFA provider (`mfaDuoResponseAllow` / `mfaDuoRequestTimeout`)